profiler/src/ProfilerEngine/Datadog.Profiler.Native/LibddprofExporter.cpp

```diff
@@ -113,9 +118,27 @@ LibddprofExporter::Tags LibddprofExporter::CreateTags(IConfiguration* configurat
     tags.Add("process_id", ProcessId);
     tags.Add("host", configuration->GetHostname());
 
-    // TODO get
-    // runtime_version
-    // runtime_platform (os and version, archi)
+    // runtime_version:
+    //    framework-4.8
+    //    core-6.0
+    std::stringstream buffer;
+    if (runtimeInfo->IsDotnetFramework())
+    {
+        buffer << "framework";
+    }
+    else
+    {
+        buffer << "core";
+    }
+    buffer << "-" << std::dec << runtimeInfo->GetDotnetMajorVersion() << "." << runtimeInfo->GetDotnetMinorVersion();
+    tags.Add("runtime_version", buffer.str());
+
+    // runtime_platform (os and version later)
+#ifdef LINUX
+    tags.Add("runtime_os", "linux");
+#else
+    tags.Add("runtime_os", "windows");
+#endif
```

Comment:
maybe add `GetOs` member function to `IRuntimeInfo` and call it from there ?

Reply:
✅ done